tool/jt.rb

```diff
@@ -737,7 +738,7 @@ def pr_clean
       puts branches_to_delete.map(&:last).map(&:to_i).to_s
 
       branches_to_delete.each do |remote_branch, _|
-        sh 'git', 'push', '--no-verify', bb, ":#{remote_branch}"
+        sh 'git', 'push', '--no-verify', *(['--dry-run'] if dry_run), bb, ":#{remote_branch}"
```

Comment:
I was thinking to just print the branches on `--dry-run`, what output does this produce?

Reply:
I tried and I think it's just better to `return` after the `puts` above.
No point to run these slow commands.

Reply:
It's left there so the user sees the command and also the result. Git prints what it would do on remote server which I think should be in the output, since `git push --no-verify --dry-run bitbucket :github/pr/153` is not immediately understandable. I don't think speed is a concern for this.

Reply:
I think it makes the command unpractical because it's so slow and does a lot of extra output. The important information is which branches are going to be deleted, not the simple git command to do it.

Reply:
I prefer it this way since it's a full dry-run not just a query to get which branches will be deleted, which conforms more the the option name. You can always kill it with `ctrl-c` if you are not interested in the git results. This is hardly an important change, it's cosmetic, the command will be rarely executed and I don't wish to spend any more time on this. I think we are just both loosing time here polishing it. If you insist on the change please do it yourself, I would suggest changing the name of the option though.

Reply:
The man page says:
```
       -n, --dry-run
           Do everything except actually send the updates.
```
What is "everything"?
I really want nothing to be changed at the git level when running `jt pr clean --dry-run`.

Reply:
I think for the purpose of this PR we can consider `git push --dry-run` safe, everything probably refers to checking with remote that data can be pushed.